openmrs/apps/registration/app.json

```diff
@@ -119,12 +119,10 @@
                 "familyName" : {"pattern" : "[a-záàãâéèêẽíìóòõôúùçA-ZÁÀÃÂÉÈÊẼÍÌÓÒÔÕÚÙÇ]{0,}", "errorMessage" : "Should contain characters"},
                 "givenName" : {"pattern" : "[a-záàãâéèêẽíìóòõôúùçA-ZÁÀÃÂÉÈÊẼÍÌÓÒÔÕÚÙÇ]{0,}", "errorMessage" : "Should contain characters"},
                 "middleName" : {"pattern" : "[a-záàãâéèêẽíìóòõôúùçA-ZÁÀÃÂÉÈÊẼÍÌÓÒÔÕÚÙÇ]{0,}", "errorMessage" : "Should contain characters"},
-                "latitude" : {"pattern" : "[a-zA-Z]{0,}", "errorMessage" : "Should contain characters"},
-                "address3" : {"pattern" : "[a-zA-Z0-9\\s]{0,}", "errorMessage" : "Should contain Characters, Numbers"},
-                "address4" : {"pattern" : "[a-zA-Z0-9\\s)-(]{0,}", "errorMessage" : "Should contain Characters, Numbers,-,(,)"},
-                "address5" : {"pattern" : "[a-zA-Z0-9\\s-]{0,}", "errorMessage" : "Should contain Characters, Numbers, -"},
-                "address6" : {"pattern" : "[a-zA-Z0-9\\s-]{0,}", "errorMessage" : "Should contain Characters, Numbers, -"},
```

Comment:
@Edgencio what is the reason for us removing address6 field from here?

Reply:
There's no specific reason! it was a mistake of mine...